pkg/nsx/services/subnetport/subnetport.go

```diff
@@ -179,11 +214,81 @@ func (service *SubnetPortService) CreateOrUpdateSubnetPort(obj interface{}, nsxS
 	if isChanged {
 		log.Info("successfully created or updated subnetport", "nsxSubnetPort.Id", *nsxSubnetPort.Id)
 	} else {
-		log.Info("subnetport already existed", "subnetport", *nsxSubnetPort.Id)
+		log.Info("Subnetport already existed", "subnetport", *nsxSubnetPort.Id)
 	}
 	return nsxSubnetPortState, enableDHCP, nil
 }
 
+// CreateDHCPStaticBinding will create DHCP static binding.
+func (service *SubnetPortService) CreateDHCPStaticBinding(staticBinding *model.DhcpV4StaticBindingConfig, subnetInfo servicecommon.VPCResourceInfo, bindingId string) error {
```

Comment:
Is the bindingId always the same as *staticBinding.Id? If so maybe we do not need to pass it again as an argument?